set.mm

```diff
@@ -353426,6 +353431,36 @@ the complex square root function (the branch cut is in the same place for
       LCMNDOKBCPQDRQBDRQCDRQPQSEFGHIBCDTUA $.
   $}
 
+  $( Square root theorem over the complex numbers for the complex power
+     function.  Theorem I.35 of [Apostol] p. 29.  Compare with ~ sqrtth .
+     (Contributed by AV, 23-Dec-2022.) $)
+  cxpsqrtth $p |- ( A e. CC -> ( ( sqrt ` A ) ^c 2 ) = A ) $=
+    ( cc wcel csqrt cfv c2 ccxp co wceq wi cc0 wa 2cnne0 0cxp ax-mp fveq2 sqrt0
+    wne syl6eq adantr oveq1d 3eqtr4a a1d cexp sqrtcl simpl simpr sqr00d necon3d
+    id ex imp cz 2z a1i cxpexpzd sqrtth eqtrd expcom pm2.61ine ) ABCZADEZFGHZAI
+    ZJAKAKIZVDVAVEKFGHZKVCAFBCFKRLVFKIMFNOVEVBKFGVEVBKDEKAKDPQSUAVEUJUBUCVAAKRZ
+    VDVAVGLZVCVBFUDHZAVHVBFVAVBBCVGAUETVAVGVBKRVAVBKAKVAVBKIZVEVAVJLAVAVJUFVAVJ
+    UGUHUKUIULFUMCVHUNUOUPVAVIAIVGAUQTURUSUT $.
+
+  ${
+    $d a b $.
+    $( There exist irrational numbers ` a ` and ` b ` such that ` ( a ^ b ) `
+       is rational.  Statement in the Metamath book, footnote 27 on page 17.
```

Comment:
Can you add the Section (which is typically more stable than the page):
```
...footnote 27 on page 17 (Section 1.1.5).
```
and similarly for your other theorems.

Reply:
I don't think the current tools let you word it quite like that, but including the section does make sense to me.